pkgs/development/python-modules/diagrams/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+, graphviz
+}:
+
+buildPythonPackage rec {
+  pname = "diagrams";
+  version = "0.17.0";
+  disabled = ! isPy3k;
```

Comment:
```suggestion
  disabled = !isPy3k;
```

Reply:
Thank you for looking at this PR SuperSandro2000 😀

Reply:
In the GH branch, I switched this to `pythonOlder` [link, see L#19](https://github.com/NixOS/nixpkgs/pull/102060/files#diff-dcdb3f0a79fc374de9f7df1479a0c343bb35c7e8c2903ac35e2e336c18b1001e)